plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactoryTest.kt

```diff
@@ -218,4 +219,29 @@ class CameraAnimatorsFactoryTest {
       }
     }
   }
+}
+
+@RunWith(Parameterized::class)
+class WrapCoordinateTest(private val actual: Double, private val expected: Double) {
+
+  private companion object {
+    @JvmStatic
+    @Parameterized.Parameters
+    fun data() = listOf(
+      arrayOf(CameraTransform.wrap(-0.911, -LONGITUDE_MAX, LONGITUDE_MAX), -0.911),
```

Comment:
nit:

you can simplify call, store only data and in test function call `CameraTransform.wrap`
for spliting actual and expected can use `Triple(-0.911, -LONGITUDE_MAX, LONGITUDE_MAX)`, `-0.911`